lambdas/file_access_patterns/lambda_for_fileUpload/index.js

```diff
@@ -38,18 +34,18 @@ exports.uploadFile = async (event) => {
     }).promise()
 
     await dynamo.update({
-    TableName: "V-Transfer",
-    Key: {
-      PK: `USER#${reqBody.user_email}`,
-      SK: `METADATA`,  //token is password hash
-    },
-    UpdateExpression: "add storage_used :file_size",
-    ExpressionAttributeValues: {
-      ":file_size": reqBody.size,
-    },
-  }).promise()
-  return response(201,"File upload success")
-}
+      TableName: "V-Transfer",
```

Comment:
use separate try/catch for each dynamo call
```
try{ dynamo.put().promise }
catch {//handle put error here }
try { dynamo.update().promise() }
catch{ //handle update error here }
```

Reply:
Okay